library/src/main/java/com/novoda/downloadmanager/lib/BatchRepository.java

```diff
@@ -86,8 +98,16 @@ int getBatchStatus(long batchId) {
             }
         }
 
+        boolean hasCompleteItems = statusCountMap.hasCountFor(DownloadStatus.SUCCESS);
+        boolean hasPendingItems = statusCountMap.hasCountFor(DownloadStatus.PENDING);
+        boolean hasSubmittedItems = statusCountMap.hasCountFor(DownloadStatus.SUBMITTED);
+        boolean hasOtherItems = statusCountMap.hasNoItemsWithStatuses(STATUSES_EXCEPT_PENDING_SUCCESS_SUBMITTED);
+        if (hasCompleteItems && (hasPendingItems || hasSubmittedItems) && !hasOtherItems) {
```

Comment:
If you have 1 complete and 2 pending items, the batch is technically running (you've just finished one download before starting the next), but the batch's status will actually be pending. 

But if it has no complete items you're at the start of the download and it is actually pending